Expensify/App
# Only trigger deploy blocker notifications on issues

### Details
We were triggering the deploy blocker notifications on PRs which could include forks, which was causing issues here: https://github.com/Expensify/App/actions/runs/1140938643

### Tests
1. Merge this PR
2. Add a `DeployBlockerCash` label to a PR, verify it does not ping in Slack
3. Add a `DeployBlockerCash` label to a issue, verify it does ping in Slack
